FriendsTown.Web/Models/Office.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace FriendsTown.Web.Models
+{
+    public class Office
+    {
+        public string Code { get; set; }
+        public string Name { get; set; }
+
+        public List<Office> GetAll() 
+        {
```

Comment:
We should think of removing this piece now that we can get data from the database. ;)